### PR TITLE
[ADD] init hook to set gender based on titles

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Same comment as in https://github.com/OCA/partner-contact/pull/279/files#r65131117

File: partner_contact_gender/__init__.py [hunk 1]
```diff
@@ -3,3 +3,4 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from . import models
+from hooks import post_init_hook
```
Comment (pedrobaeza) [inline partner_contact_gender/__init__.py]: `from .hooks import post_init_hook`

**Comment by hbrunn** [comment]
@pedrobaeza thanks, done

**Comment by pedrobaeza** [comment]
:+1: 

Thanks

**Comment by amkarthik** [comment]
+1

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**